sigs/cncf-sigs.md

```diff
@@ -0,0 +1,252 @@
+
+        * As far as possible, information should be based on research and fact gathering, rather than pure marketing or speculation.  
+
+#### End User Input Gathering (Inbound Communication)
+
+    * Gather useful end user input and feedback regarding expectations, pain points, primary use cases etc.
+
+    * Compile this into easily consumable reports and/or presentations to assist projects with feature design, prioritization, UX etc.
+
+#### Community Enablement
+
+    * SIGs are open organizations with meetings, meeting agendas and notes, mailing lists, and other communications in the open
+
+    * The mailing list, SIG meeting calendar, and other communication documents of the SIG will be openly published and maintained
+
+#### As Trusted Expert Advisors to the TOC
+
+    * Perform technical due diligence on new and graduating projects, and advise TOC on findings.
+
+    * Be involved with, or periodically check in with projects in their area, and advise TOC on health, status and proposed actions (if any) as necessary or on request.
+
+#### SIG Charter:
+
+    * formally reviewed annually, and approved by the TOC.  The charter must clearly articulate:
+
+        * what is in and out of scope of the SIG
+
+        * whether and how it overlaps and interfaces with other CNCF SIG’s or other relevant groups.
+
+        * how it operates and is governed, and specifically whether and how it deviates from standard SIG operating guidelines provided by the TOC.  Deviation from these guidelines is discouraged, unless there are good and well-documented reasons for such divergence, approved by the TOC. 
+
+See [Example Responsibilities of a CNCF SIG](https://docs.google.com/document/d/1L9dJl5aBFnN5KEf82J689FY0UtnUawnt9ooCq8SkO_w/edit?usp=sharing).
+
+## Operating Model
+
+*Important*
+
+*Each SIG is supported by a named member of the CNCF executive staff who is accountable for liaison with the CNCF Exec, plus communication and performance of the SIG, with quarterly and annual reporting to GB & TOC. *** **
+
+As a starting point let’s be inspired by CNCF OSS Projects and by K8S SIGs.  That means minimal viable governance and community-based organisation.
+
+### SIG Formation, Leadership and Membership Composition
+
+1. SIGs are formed by the TOC.  Initial SIGs are listed below, and will be adapted over time as required.  If members of the community believe that additional SIGs are desired, they should propose these to the TOC, with clear justification, and ideally volunteers to lead the SIG. The TOC wishes to have the smallest viable number of SIGs, and for all of them to be highly effective (as opposed to a "SIG sprawl" with large numbers of relatively ineffective SIGS).
+
+2. SIG has three co-chairs, who are TOC Contributors and recognised as unbiased experts in that area.
+
+3. SIG has one TOC liaison who is a voting member of the TOC acting as non-exec chair on occasions when TOC input is deemed necessary by the TOC or the SIG chairs
```

Comment:
Are they one of the 3 co-chairs or are they a 4th?  What is a "non-exec chair" vs a "normal chair" ? Can we define what we mean by that?

Do we expect them to be very active within the SIG or just act as the liaison to the TOC? I'm asking because if they're more of a liaison then I'm not sure why we're making them a chair - I think chairs need to be very active.

Reply:
@duglin 4th.  I think the second half of the sentence makes the intention sufficiently clear? "on occasions when TOC input is deemed necessary by the TOC or the SIG chairs"?

i.e. the intention is that the 3 co-chairs run the show, with the (4th) TOC liason non-exec chair assisting under specific circumstances when requested to do so.

Do you think we need to add more clarity than what is already there?

Reply:
Yup I do because I can interpret this as.... we have a 4th chair that could choose to do nothing until TOC input is needed and then they wake up and engage. Why? Answer: To talk to the TOC about some issue that they haven't been paying attention to. :-)  Seems odd that the other 3 chairs can't talk directly to the TOC if they need their input. I'm not in favor of roles that appear to serve no real purpose. I'm assuming the other chairs are able to speak for themselves to the TOC.

I still don't know what a "non-exec" chair is.

Reply:
@duglin To be clear, the intention is that the TOC liason effectively represents the TOC on the SIG.  That would require that they be sufficiently engaged.  The example you sketch is of an insufficiently engaged TOC liason, who would presumably either not have been appointed by the TOC in the first place, or have been kicked out by the TOC (possibly at the request of the chairs), for not doing their job properly.

Reply:
@duglin If removing the words "non-exec chair" solves this problem, I think that's fine.  The main point is that a TOC member is appointed as point person for each SIG.

Reply:
I think removing non-exec would be good since I don’t think it’s clear what that means.

As for this TOC rep/chair, you seem to be implying that this person speaks for the TOC which I’m not comfortable with since I don’t think the TOC has a single voice/opinion. I would really prefer that if the SIG wants the TOC’s input that they just ask the TOC directly